packages/mwp-app-render/src/util/seo/metaTags.jsx

```diff
@@ -0,0 +1,150 @@
+};
+
+export const generateMetaTags = tags =>
+	tags.map((meta, index) => <meta {...meta} key={`meta-${index}`} />);
+
+/**
+ * Generates array of basic common meta data for all pages
+ * @param {String} options.title title for the current page
+ * @param {String} options.description description for the current page
+ * @param {String} options.ogTitle title for og:title
+ * @param {String} options.ogDescription title for og:description
+ * @param {String} options.ogImageUrl path to image for og:image
+ * @param {String} options.twitterImageUrl path to twitter image for sharing
+ * @param {String} options.keywords keywords for the current page
+ * @param {String} options.baseUrl base url of the current page
+ * @param {String} options.url path of the current page
+ * @param {String} options.appPath current pages path in apps
+ * @return {Array} array of meta objects or use by Helmet
+ */
+export const generateMetaData = ({
+	title,
+	description,
+	ogTitle,
+	ogDescription,
+	imageUrl,
+	keywords,
+	baseUrl,
+	route,
+	appPath,
+	localeCode,
+}) => {
+	const desc = stripAndTruncate(description);
+	const ogDesc = stripAndTruncate(ogDescription);
+
+	return [
+		{ name: 'description', content: desc },
+		{ name: 'keywords', content: keywords },
+		{ property: 'fb:app_id', content: FB_APP_ID },
+		{ property: 'og:site_name', content: 'Meetup' },
+		{ property: 'og:type', content: 'article' },
+		{ property: 'og:title', content: ogTitle || title },
+		{ property: 'og:description', content: ogDesc || desc },
+		baseUrl && {
+			property: 'og:url',
+			content: generateCanonicalUrl(baseUrl, localeCode, route),
+		},
+		{ property: 'og:image', content: imageUrl },
+		{ property: 'al:android:app_name', content: 'Meetup' },
+		{ property: 'al:android:package', content: 'com.meetup' },
+		{ property: 'al:ios:app_store_id', content: TWITTER_APP_ID },
+		{ property: 'al:ios:app_name', content: 'Meetup' },
+		{ property: 'al:web:should_fallback', content: 'true' },
+		appPath && { property: 'al:android:url', content: appPath },
+		appPath && { property: 'al:ios:url', content: appPath },
+		baseUrl &&
+			route && {
+				property: 'al:web:url',
+				content: generateCanonicalUrl(baseUrl, localeCode, route),
+			},
+		{ property: 'twitter:card', content: 'summary_large_image' },
+		{ property: 'twitter:site', content: '@meetup' },
+		{ property: 'twitter:image', content: imageUrl },
+		{ property: 'twitter:description', content: desc },
+		{ property: 'twitter:app:id:iphone', content: TWITTER_APP_ID },
+		{ property: 'twitter:app:name:iphone', content: 'Meetup' },
+		{ property: 'twitter:app:id:ipad', content: TWITTER_APP_ID },
+		{ property: 'twitter:app:name:ipad', content: 'Meetup' },
+		{ property: 'twitter:app:id:googleplay', content: 'com.meetup' },
+		{ property: 'twitter:app:name:googleplay', content: 'Meetup' },
+	].filter(metaEntry => metaEntry !== undefined && metaEntry !== '');
```

Comment:
is this the same as `].filter(x => x);` ? You want any truthy value to pass?

Reply:
Pretty sure I didn't write this bit but I think its aim is to not render any 'empty meta tags like - `<meta property="description" content="" />`. So I think what you're suggesting will work. I'll test.